locustfiles/lifecycle.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""
+This example shows the various ways to run things before/outside the normal task execution
+flow, which is very useful for fetching test data.
+
+This was copied from https://github.com/locustio/locust/blob/master/examples/test_data_management.py
```

Comment:
It looks like both our repo and theirs are MIT licensed, maybe we should include their copyright as well in this file https://github.com/locustio/locust/blob/master/LICENSE

Reply:
Tbh, I don't know much about the license stuff. Do you mean like include a link to their license? Or add a copyright statement?

Reply:
I would say add the copyright statement to the top of that file if that's what you copied here.